app.rb

```diff
@@ -0,0 +1,84 @@
+require './book'
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    puts 'Books: '
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+    puts ''
+  end
+
+  def list_people
+    puts 'People: '
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    puts ''
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)?'
+    print '[Input the number]: '
+    person_type = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Age : '
+    age = gets.chomp.to_i
+    case person_type
+    when 1
+      print 'Has parent permission? [Y/N]: '
+      permission = gets[0]
+      permission = (permission == ('Y' || 'y'))
+      @people << Student.new('Unknown', age, name, permission)
```

Comment:
- [ ] `required`
- [ ] Amazing work so far 👏 
- [ ] However there is a few things which need adjustment
- [ ] I'm able to create a person its giving an error
- [ ] And also the u.i is supposed to welcome message and the instructions.
- [ ] however on top of the welcome its showing some things which are not supposed to be there.
- [ ] Please fix it according to the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- [ ] And see the screenshot below and highlighted areas.

![image](https://user-images.githubusercontent.com/66230306/201495649-8c9d21db-7373-4b29-9270-e58d55b801f0.png)